### DISCUSSION TITLE
Bookmarks or quick links in new tab

### DISCUSSION THREAD

**Discussion by nmhillusion** [dailydotdev/daily#397] [Feature Requests]
In the first part of new tab, in frequently visitted pages, I think it will be better if able to customize links and manage what links will be displayed there.

**Comment by idoshamun** [comment]
Hi @nmhillusion,

It's already on our roadmap :)

**Reply by nimrodkra** [reply]
@nmhillusion It's ready, give it a try! https://app.daily.dev/posts/rIZ_uqa1Q

**Reply by nmhillusion** [reply]
> @nmhillusion It's ready, give it a try! https://app.daily.dev/posts/rIZ_uqa1Q

Has it not updated on Edge browser yet?

**Reply by idoshamun** [reply]
Should be available now
